packages/cashier/src/Stores/Cashier/cashier-store.js

```diff
@@ -1065,18 +1065,18 @@ export default class CashierStore extends BaseStore {
             }
         }
 
-        const mt5_login_list = (await this.WS.storage.mt5LoginList())?.mt5_login_list;
+        const mt5_login_list = (await this.WS.storage.mt5LoginList())?.mt5_login_list || [];
         // TODO: remove this temporary mapping when API adds market_type and sub_account_type to transfer_between_accounts
         const accounts = transfer_between_accounts.accounts.map(account => {
-            if (account.account_type === 'mt5') {
+            if (account.account_type === 'mt5' && Array.isArray(mt5_login_list) && mt5_login_list.length) {
```

Comment:
I think `mt5_login_list` is always an array now. Am I wrong?
```suggestion
            if (account.account_type === 'mt5' &&  mt5_login_list.length) {
```

Reply:
oh seems to duplicate!